src/gnssro/gnssro_bufr2ioda.f90

```diff
@@ -311,6 +311,7 @@ program gnssro_bufr2ioda
 call check( nf90_def_dim(ncid, 'nvars', nvars,   nvars_dimid) )
 call check( nf90_def_dim(ncid, 'ndatetime', ndatetime,   ndatetime_dimid) )
 call check( nf90_put_att(ncid, NF90_GLOBAL, 'date_time', anatime_i) )
+call check( nf90_put_att(ncid, NF90_GLOBAL, '_FillValue', missingvalue) )
```

Comment:
The _FillValue attribute needs to be added to each variable that contains missing values. It won't have an effect as a global attribute.

One quick way to test if you've got it right is to run ncdump on the output file. If a variable with value equal to missingvalue shows up with a "-" in the output of ncdump, then the _FillValue attribute is working as intended. If instead the value shows up as what you would expect for missingvalue (-9.9e15) then it is not being recognized as the fill value.

Reply:
got it! Thank you very much!

Reply:
Just to clarify on @srherbener's description.  The locations equal to _FillValue show up as "_", an underscore, not a hyphen.

Reply:
I have added _FillValue to refractivity as well as many other attributes. Please review and try @srherbener @jjguerrette @svahl991 
Now I see the "_"s in ncdump.